ssn/integrated/sosa.ttl

```diff
@@ -452,11 +453,10 @@ sosa:Observation a rdfs:Class , owl:Class ;
     rdfs:isDefinedBy sosa: .
 
 
-
-sosa:Actuation a rdfs:Class , owl:Class ;
+sosa:Actuation a owl:Class ;
   rdfs:label "Actuation"@en ;
-  skos:definition "An Actuation carries out an (Actuation) Procedure to change the state of the world using an Actuator."@en ;
-  rdfs:comment "An Actuation carries out an (Actuation) Procedure to change the state of the world using an Actuator."@en ;
+  skos:definition "An Actuation carries out an (Actuation) Procedure to change the state of the world using an Actuator. hasResult indicates the intended value of the ActuatableProperty at the completion of the Actuation"@en ;
```

Comment:
> at the completion of the Actuation

at some point in the future? When changing a thermostat It may take hours before the temperature has reached the desired value. At the completion of the Actuation the heating/cooling system is only _starting_ to change the temperature.

Reply:
There are many complexities in specific actuation situations that are beyond the scope of this general framework. However, there is one common requirement across all actuations: the target state i.e. the intended value of the actuatable property. That is how I understood it. 

More complex scenarios must be modeled in applications that extend or specialize SSN/SOSA.